[NO-ISSUE] refactor: cache settings + edge caching (#704)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/build/edge-application/cache-settings.mdx

## Slice Settings

**Slice** works with a default slice range of *1024 kB*. If you want to modify the slice range, [contact the Sales team](https://www.azion.com/en/contact-sales/).

> 

Author: ```suggestion

<Button href="/en/documentation/products/edge-application/edge-caching/#slice-files" text="Learn more about slice files"></Button>

```

> ## Slice Settings

Reviewer: ```suggestion
## Slice settings
```

Author: É o nome da seção na UI, tem que ser assim D:

**Slice** works with a default slice range of *1024 kB*. If you want to modify the slice range, [contact the Sales team](https://www.azion.com/en/contact-sales/).

> <Button href="/en/documentation/products/edge-application/edge-caching/#slice-files" text="Learn more about slice files"></Button>

Reviewer: ```suggestion
<Button href="/en/documentation/products/edge-application/edge-caching/#slice-files" text="go to slice files documentation"></Button>
```

## Slice Settings

> **Slice** works with a default slice range of *1024 kB*. If you want to modify the slice range, [contact the Sales team](https://www.azion.com/en/contact-sales/).

Reviewer 2: ```suggestion
**Slice** works with a default slice range of *1,024 kB*. If you want to modify the slice range, [contact the Sales team](https://www.azion.com/en/contact-sales/).
```
